src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -79,28 +77,13 @@ public void c024928()
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
 
+#if !DEBUG 
```

Comment:
Rather than use a conditional directive here, can we introduce `FactReleaseOnly`?  A custom `Fact` type is generally how we control tests that execute under specific conditions.